blogs/2023/06/05/vscode-wasm-wasi.md

```diff
@@ -179,9 +179,9 @@ What we do know is that we will further explore the following topics:
 
 * The WASI team is working on a preview2 and preview3 of the specification, which we plan to support as well. The new versions will change the way a WASI host is implemented. However, we are confident that we can keep our API, which is exposed in the WASM execution engine extension, mostly stable.
 * There is also the [WASIX](https://wasix.org/) effort that extends WASI with additional [operating system-like features](https://wasix.org/docs/api-reference) such as process or futex. We will continue to watch this work.
-* Many language servers for VS Code are implemented in languages different than JavaScript or TypeScript. We plan to explore the possibility of compiling these language servers to `wasm32-wasi` and running them in VS Code for the Web as well.
+* Many language servers for VS Code are implemented in languages different from JavaScript or TypeScript. We plan to explore the possibility of compiling these language servers to `wasm32-wasi` and running them in VS Code for the Web as well.
 * Improving debugging for Python on the Web. We have started to work on this, so stay tuned.
-* Add support so that extension B can run WebAssembly code contributed by extension A. This will, for example, allow arbitrary extensions to execute Python code by reusing the extension that contributed the Python WebAssembly.
+* Add support so that extension B can run WebAssembly code contributed by extension A. This will, for example, allow arbitrary extensions to execute Python code by reusing the extension that contributed to the Python WebAssembly.
```

Comment:
This sentence is correct. The extension has contributed the entire Python WebAssembly

Reply:
Thanks for the clarity, appreciate that!